### ISSUE TITLE
About VOCA Metric calculation

### ISSUE THREAD

**Issue by Rookie-Kai** [psyai-net/SelfTalk_release#4]
Hi ZiQiao, thank you so much for your work
I found in your paper that you have calculated LVE on VOCA-test dataset,since I found that after using FLAME_masks.  pkl to get the lip vertices, the results calculated by GT are not quite the same as yours, I suspect that there is a problem with my processing could you please provide your testing process?
Thank you again for your work

**Comment by ZiqiaoPeng** [comment]
We use the metric calculation code from [CoderTalker](https://github.com/Doubiiu/CodeTalker/blob/main/main/cal_metric.py). During the comparison you need to subtract the value of the template. The lip vertex error of our released pre-training model should be 2.8, which is better than the results reported in our paper.

**Comment by Rookie-Kai** [comment]
Thank you very much for your busy reply, but when I used the evaluation code of Codetalker, the value of each model calculated by me was different from that in your paper. I don't know what specific changes need to be made in the application of vocaset test set. If convenient, could you please provide a copy of the test code you used at that time? If it is not convenient to make public, please send it to my email address, which is
aaaaaki2023@gmail.com

This problem has been bothering me for a long time, and I'm sorry to bother you all the time

**Comment by AlongLove** [comment]
@Rookie-Kai @ZiqiaoPeng I would like to ask you if you have encountered non-convergence in training meshtalk on vocaset and how you solved it? And please don't hesitate to tell me, thanks!

**Comment by hrWong** [comment]
Hi @Rookie-Kai Have you solved this problem and can you tell me how you solved it, I'd appreciate it!
